frontend/src/CreateBook.tsx

```diff
@@ -2,6 +2,7 @@ import React, {FormEvent} from "react";
 import Modal from 'react-modal';
 import axios from "axios";
 import {BookState} from "./BookState";
+import "./css/BookOverview.css";
```

Comment:
If you need the css from your BookOverview component for the CreateBook component consider creating a new css-file for CreateBook with only the css, that is used there